grpc/src/main/java/com/linecorp/armeria/common/grpc/GrpcExceptionHandlerFunction.java

```diff
@@ -48,12 +48,13 @@ static GrpcExceptionHandlerFunction of() {
     }
 
     /**
-     * Maps the specified {@link Throwable} to a gRPC {@link Status},
-     * and mutates the specified {@link Metadata}.
-     * If {@code null} is returned, the built-in mapping rule is used by default.
+     * Maps the specified {@link Throwable} to a gRPC {@link Status} and mutates the specified {@link Metadata}.
+     * The {@link Status} that was created via {@link Status#fromThrowable(Throwable)} might be specified
+     * as well. You can return this {@link Status} or any other {@link Status} as needed.
+     * If {@code null} is returned, {@link #of()} will be used as the default.
```

Comment:
Perhaps we should document when `null` is specified as `status`? I'm curious what a user should return when the specified `status` is `null`.

Reply:
Thanks! Updated. 😉